_rules/zoom-text-no-overflow-clipping-59br37.md

```diff
@@ -0,0 +1,249 @@
+---
+id: 59br37
+name: Zoomed text node is not clipped with CSS overflow
+rule_type: atomic
+description: |
+  This rule checks that text nodes are not unintentionally clipped by overflow, when a page is zoomed to 200% on 1280 by 1024 viewport;
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:1.4.4: # Resize Text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Wilco Fiers
+  citations:
+    - The Raven, poem by Edgar Allan Poe
+---
+
+## Applicability
+
+The rule applies to any [text node][] for which all of the following is true when in a [viewport size][] of 640 by 512:
+
+- The [text node][] is [visible][]; and
+- The [text node][] has an HTML element as a [parent][] in the [flat tree][]; and
+- The [text node][] has an [ancestor][] in the [flat tree][] with a [computed][] [overflow-x][overflow] or [overflow-y][overflow] of `hidden` or `clip`; and
+- The [text node][] does not have an [ancestor][] in the [flat tree][] with an `aria-hidden` attribute set to `true`
```

Comment:
I don't understand why this condition is needed. The SC and the rule seems to be concerned with sighted users more than AT users, and they will essentially make no difference between `aria-hidden` subtrees and the rest.

Why is it "OK" to clip a text if it is `aria-hidden`?

Reply:
This is to avoid false positives. I'll rephrase the second assumption to make this clearer.

Reply:
I fail to find a false positive caused by `aria-hidden="true"` 😞 
Can you provide me one?

Reply:
For example a site that uses ASCII art for something decorative. If it's decorative, SC 1.1.1 requires it be marked up with aria-hidden=true, so that it can be ignored by AT. Since it's decorative, it matter if it's clipped, so keyboard accessibility isn't necessary either.